packages/FormElement/src/FormElement.styles.js

```diff
@@ -8,18 +8,24 @@ const FontSizes = {
 };
 
 export const SectionsContainer = styled.div`
```

Comment:
why are yo not following the function format? I think this should be also changed isn't'?